Libraries/ReactIOS/YellowBox.js

```diff
@@ -82,9 +84,29 @@ function updateWarningMap(format, ...args): void {
     ...args.slice(argCount).map(stringifySafe),
   ].join(' ');
 
-  const count = _warningMap.has(warning) ? _warningMap.get(warning) : 0;
-  _warningMap.set(warning, count + 1);
+  var warningInfo = _warningMap.get(warning);
```

Comment:
Thanks for cleaning up this ugly ternary, but in the future please use let/const to match the rest of the file.

@frantic: What do you think about adding that to our OSS lint since we just added it to internal lint? Will it be too annoying in PRs?